specification/configuration.md

```diff
@@ -133,7 +133,10 @@ beyond the OpenTelemetry specification exist.
 - Zipkin exporter
   - Distribution MUST NOT list Zipkin exporter as supported (not supported by Smart Agent)
 - `OTEL_TRACES_EXPORTER`
-  - Non-RUM distribution MUST default to `otlp` over gRPC with an endpoint of `localhost:4317`
+  - Non-RUM distribution MUST support following, additional values:
+    - `otlp-grpc` as an alias for `OTEL_TRACES_EXPORTER=otlp` and  `OTEL_EXPORTER_OTLP_PROTOCOL=grpc`
+    - `otlp-http` as an alias for `OTEL_TRACES_EXPORTER=otlp` and  `OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf`
```

Comment:
This seems to be introducing an alternate way to specify the exporter than what is present in OpenTelemetry. Here it is specified as a single value, in OpenTelemetry it is specified as two. They both achieve the same thing though. By introducing an alternate way to specify this exporter Splunk's use of OpenTelemetry will diverge.

Instead of creating our own way to define these exporters, I think the better approach is to say projects need to support the `OTEL_EXPORTER_OTLP_PROTOCOL` environment variable. That way we will remain in-line with OpenTelemetry and still achieve what this PR is aiming to accomplish.